src/commands/createFunction/scriptSteps/NodeProgrammingModelFunctionCreateStep.ts

```diff
@@ -0,0 +1,38 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzExtFsExtra, nonNullProp } from '@microsoft/vscode-azext-utils';
+import * as path from 'path';
+import { functionSubpathSetting, ProjectLanguage } from '../../../constants';
+import { IScriptFunctionTemplate } from '../../../templates/script/parseScriptTemplates';
+import { getWorkspaceSetting } from '../../../vsCodeConfig/settings';
+import { FunctionCreateStepBase } from '../FunctionCreateStepBase';
+import { IScriptFunctionWizardContext } from './IScriptFunctionWizardContext';
+import { getScriptFileNameFromLanguage } from './ScriptFunctionCreateStep';
+
+export class NodeProgrammingModelFunctionCreateStep extends FunctionCreateStepBase<IScriptFunctionWizardContext> {
```

Comment:
> Any name/file with NodeProgrammingModel... is because I wasn't creative enough to think of anything better. Is there a better way that I should denote this? Maybe just NodejsV4...?

I would go with `NodeV4`